web/src/util.ts

```diff
@@ -458,6 +457,10 @@ export function get_remaining_time(start_time: number, duration: number): number
 // Helper for shorthand for Typescript to get an item from a list with
 // exactly one item.
 export function the<T>(items: T[] | JQuery<T>): T {
-    assert.equal(items.length, 1, "the: expected exactly one item");
+    if (items.length === 0) {
+        blueslip.error("the: expected only 1 item, got none");
+    } else if (items.length > 1) {
+        blueslip.error("the: expected only 1 item, got more");
```

Comment:
Does this expose to blueslip the actual number of items? I guess it'll almost always be 2, but still, seems annoying to not specify.

Reply:
I can specify if you think that would be helpful! I'm not familiar enough with blueslip to know if that would make the collecting of errors less clean or something. I'll add it now :)

Reply:
Ah I found a way to add the number of items without changing the specific error message, so I did that. Ready for you again!